bgpd/bgp_io.c

```diff
@@ -336,7 +336,24 @@ static uint16_t bgp_write(struct peer *peer)
 				BGP_EVENT_ADD(peer, TCP_fatal_error);
 				SET_FLAG(status, BGP_IO_FATAL_ERR);
 			} else {
-				SET_FLAG(status, BGP_IO_TRANS_ERR);
+				uint32_t last_write;
+				uint32_t length;
```

Comment:
I think this variable name is confusing, but this is a nit -- i.e., you length of time not len of string/array/etc, Maybe duration is better.  This is a non-blocking comment